src/rules/jsxUseTranslationFunctionRule.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * @license
+ * Copyright 2017 Palantir Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Lint from "tslint";
+import * as ts from "typescript";
+import {nodeIsKind} from "../guards";
+
+export class Rule extends Lint.Rules.AbstractRule {
+    public static TRANSLATABLE_ATTRIBUTES = new Set(["placeholder", "title", "alt"]);
```

Comment:
Purely syntactic checks will not be enough to fully enforce such a lint rule, but I think this is good enough to be merged in. For example, our `Button` components in https://github.com/palantir/blueprint use a `text: string` attribute which will not be linted by this rule.